### PR TITLE
Ewl 3364 hub hero side

### PR DESCRIPTION
**Pull request by aCyborg**
To Test go to Molecules > Components > 'Hub Hero Side' and 'Hub Hero Side Left'

### CODE REVIEW

**Comment by froboy** [comment]
@aCyborg for now use disks and numbers. I think that's going to be the decision.

**Comment by aCyborg** [comment]
Confirmed with Andy to leave discs and numbers

File: styleguide/source/_patterns/01-molecules/06-components/10-hub-hero/_hub-hero.scss [hunk 1]
```diff
@@ -44,7 +45,7 @@
 
 .hub-hero_title {
   background-color: rgba($white, .45);
-  padding: 20px 20px 0;
+  padding: 20px 20px 8px;
```
Comment (froboy) [inline styleguide/source/_patterns/01-molecules/06-components/10-hub-hero/_hub-hero.scss]: This effects all of the hub hero patterns. It's not a problem, just making sure it's intentional.

File: styleguide/source/_patterns/01-molecules/06-components/10-hub-hero/_hub-hero.scss [hunk 2]
```diff
@@ -83,3 +84,53 @@
...
+    padding: 20px;
+    margin-top: auto;
+
+    .button-small {
+      width: 100%;
+    }
+  }
+}
+
+.hub-hero-side-left {
```
Comment (froboy) [inline styleguide/source/_patterns/01-molecules/06-components/10-hub-hero/_hub-hero.scss]: Seems like Side Left actually aligns the content right... Is this right? How would you explain it?

File: styleguide/source/_patterns/01-molecules/06-components/10-hub-hero/_hub-hero.scss (hunk 1 above)
Comment (aCyborg) [inline styleguide/source/_patterns/01-molecules/06-components/10-hub-hero/_hub-hero.scss]: intentional!

**Comment by froboy** [comment]
LGTM. Thanks! Merging this to push up to gh-pages, but ticket will be pending review from Andy.

**Pull request merged**